src/search/csearch.c

```
@@ -135,26 +139,64 @@ int main(int argc, char *argv[])
           #ifdef ERROR
           fprintf(STDERR,
-             "*** error (SEARCH): unknown search type \"%s\" (option -s)\n", 
+             "*** error (SEARCH): unknown search type \"%s\" (option -s)\n",
              argv[i_arg]);
           #endif
           exit(1);
         }
         
       } /* search type */
+
+      if (strcmp(argv[i_arg], "--max-evals") == 0)
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)